### PR TITLE
[zh]Update README-zh.md

### PR DESCRIPTION
**Pull request by HongjiangHuang**

### CODE REVIEW

**Comment by HongjiangHuang** [comment]
cncf-cla done.

File: README-zh.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ You can also reach the maintainers of this project at:
 
 你也可以通过以下渠道联系本项目的维护人员：
 
-- [Slack](https://kubernetes.slack.com/messages/sig-docs)
+- [Slack](https://kubernetes.slack.com/messages/sig-docs) [加入Slack](https://slack.k8s.io/)
```
Comment (tengqm) [inline README-zh.md:187]: line 177 should be fixed as well.

Comment (HongjiangHuang) [reply README-zh.md:187]: fixed.

**Review by onlydole — APPROVED** [review]
Thank you, @HongjiangHuang

**Pull request merged**